lib/wasi/src/os/command/builtins/cmd_wasmer.rs

```diff
@@ -0,0 +1,154 @@
+use std::{any::Any, ops::Deref, sync::Arc};
+
+use wasmer::{FunctionEnvMut, Store};
+use wasmer_vbus::{BusSpawnedProcess, SpawnOptions, VirtualBusError};
+use wasmer_wasi_types::wasi::Errno;
+
+use crate::{
+    bin_factory::{spawn_exec, BinaryPackage, ModuleCache},
+    syscalls::stderr_write,
+    VirtualTaskManager, VirtualTaskManagerExt, WasiEnv, WasiRuntimeImplementation,
+};
```

Comment:
but for that to work, we would need hidden syscall probably (like to fetch webc packages)

Reply:
I don't think we want to push the Wasmerception too far.

It would be nice to have compilation etc available inside Wasmer, so for this having a published Wasmer package could make sense.

But for downloading the webc we  want the environment (aka Deploy or the browser)  to do it for caching. 
And running it must be done by the environment anyway. 

So we could have an "embedded Wasmer" build that does everything except support "run", but I believe that's quite a bit off, right?

For now I will move the custom command to a `wasmer-utils` crate, so it can be used from there for Deploy and `wasmer-web` and the CLI.

Reply:
Tracked in #3606